docs/dg/dev/ai-assistants-getting-started-guide.md

```diff
@@ -0,0 +1,128 @@
+---
+title: AI-Assistants getting started guide
+description: This is a step-by-step checklist that you can follow through all the stages of working with AI tools in Spryker.
+template: concept-topic-template
+---
+
+This document provides an overview of the AI Assistants feature in Spryker Cloud Commerce OS. It explains how to set up and configure AI Assistants in your Spryker project. Also, it provides basic examples how to use different tools.
+
+## Important information
+
+Before using any AI-related tool, please consult with the legal department of your organization.
+
+## Install AI Assistants
+
+Spryker Commerce OS could be used with a lot of different tools, however our team recommends to start with the following AI Assistants:
+- GitHub Copilot
+- Qodo
+- Cursor
+
+### Install GitHub Copilot
+
+GitHub Copilot is an AI pair programmer that helps you write code faster. It draws context from comments and code, and suggests individual lines and whole functions instantly.
+To start using GitHub Copilot, you need to perform few steps:
+- Install the GitHub Copilot [extension](https://plugins.jetbrains.com/plugin/17718-github-copilot) in your IDE.
+- Authorize yourself as a GitHub user in the Preferences->Languages and Frameworks-> GitHub Copilot.
+
+
+
+
+And after these small steps, you will be able to use the GitHub Copilot.
+
+### Install Qodo
+
+The setup of the Qodo tool is straightforward. All you need to do is install the following [plugin](https://plugins.jetbrains.com/plugin/21206-qodo-gen-formerly-codiumate-). If the installation was correct, you could check that the plugin works. An example is in the screenshot below.
+
+
```

Comment:
![CleanShot 2024-10-15 at 16 36 13@2x](https://github.com/user-attachments/assets/80816df4-012b-4312-b072-9422a1d28aec)

Reply:
Does this screenshot show Codiumate? Does it always show up in this menu, regardless of what is highlighted?

Reply:
If codiumate menu exists - it means that plugin is installed successsfully